README.md

```diff
@@ -1,9 +1,41 @@
 Cloudinary Media Editing SDK
-=========================
+============================
+
 ## About
 The Cloudinary Media Editing SDK allows you to quickly and easily integrate your application with Cloudinary.
 Effortlessly edit and deliver your cloud's assets.
 
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Media Editing SDK Guide](https://cloudinary.com/documentation/media_editing_api_sdks).
+
+
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Create Media Source](#Create Media Source)
```

Comment:
to link correctly these need to look like the following:
    - [Create Media Source](#Create-Media-Source)